deno.json

```diff
@@ -1,9 +1,13 @@
 {
+  "name": "@deno/setup-deno",
+  "description": "Set up Deno easily in GitHub Actions",
+  "version": "2.0.3",
+  "license": "MIT",
+  "exports": "./src/cache.ts",
```

Comment:
The `exports` property was needed since I added the `name` property with `@deno/setup-deno` to mimic `package.json`. Though both can be removed preferable.